R/highlevel64.R

```diff
@@ -1355,6 +1355,9 @@ optimizer64 <- function(nsmall=2L^16L,
 #' @keywords manip logic
 #' @export
 match.integer64 <- function(x, table, nomatch = NA_integer_, nunique=NULL, method=NULL, ...) {
+  # trivial cases
+  if (!length(x)) return(integer())
+  if (!length(table)) return(rep(as.integer(c(nomatch[[1L]], NA_integer_)[1L]), length(x)))
```

Comment:
It looks like the intention here is to allow `nomatch=NULL` to work somehow?

But `base::match()` doesn't do that AFAICT:

https://github.com/r-devel/r-svn/blob/4d30c61ead47113adf66781ee8394ba3666a8853/src/main/unique.c#L1346-L1352

So let's just match the {base} behavior. Maybe I'm misunderstanding what `c(nomatch[1], NA)[1]` is meant to do.